scripts/ci/install-minikube.sh

```diff
@@ -57,22 +62,24 @@ uninstall() {
 
 stop() {
 	check_minikube
-	sudo $(which minikube) delete || true
-	sudo rm -rf ~/.minikube ~/.kube
+	sudo -E minikube delete || true
+	sudo rm -rf $HOME/.minikube $HOME/.kube
+	sudo rm -rf /root/.minikube /root/.kube
+	sudo systemctl stop localkube || true
 }
 
 start() {
 	check_minikube
-	sudo -E CHANGE_MINIKUBE_NONE_USER=true $(which minikube) \
-		--vm-driver=${MINIKUBE_VM_DRIVER} \
-		${MINIKUBE_FLAGS} start
-	sudo $(which minikube) status
+       	sudo -E minikube start ${MINIKUBE_FLAGS}
 	kubectl config use-context minikube
+	sudo cp -ar $HOME/.kube /root/.kube
```

Comment:
same here, why copying the config to root ?

Reply:
Our installation creates a working kubectl pointing to minikube context within /home/vagrant, so that it works for root we need to copy over ~/.kube directory